doc/usermanual/darkroom/modules/color/colorbalance.xml

```diff
@@ -65,6 +65,10 @@
 
       </itemizedlist>
 
+      By clicking on the shadows label different layouts of the controls
```

Comment:
I propose a phrasing like this:
< By clicking on the shadows label different layouts of the controls are proposed. 
> Three different control layouts can be accessed by clicking on the shadows label: